### PR TITLE
Clarify messages

### PR DESCRIPTION
**Pull request by notkarol**
* Control -> Action and State -> Controller
* timestamps now in uint64 nanoseconds
* revamped colors in labeler a bit with message changes from Label -> Quality
* Added index to Camera/Imu and removed some fields that are in config or too sensor-specific
* Modularized joystick, imu, and camera to make run simpler and use the return to exit the program upon sensor fail
* Fixed some known issues with install (evdev and pip packages needing to be installed sequentially)

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -83,3 +80,5 @@ fi
 
 # Install the derp python package
 python3 setup.py install --user
+
+echo "Please reboot your machine"
```
[the remark this answers is not in the document]
Comment (notkarol) [reply install.sh]: Yes, but then it's unclear if anything fails as we don't log by default.

File: derp/servo.py [hunk 2]
```diff
@@ -77,5 +81,6 @@ def run(self):
 
 def run(config):
```
[the remark this answers is not in the document]
Comment (notkarol) [reply derp/servo.py]: Done, called it loop.

File: capnp/messages.capnp [hunk 3]
```diff
@@ -1,67 +1,54 @@
...
-  y @7 :Float32;
-  z @8 :Float32;
-  height @9 :UInt16;
-  width @10 :UInt16;
-  depth @11 :UInt8;
-  hfov @12 :Float32;
-  vfov @13 :Float32;
-  fps @14 :UInt8;
-  jpg @15 :Data;
+  timeCreated @0 :Float64;
```
[the remark this answers is not in the document]
Comment (notkarol) [reply capnp/messages.capnp]: Agreed, as discussed offline. Reverting

File: derp/joystick.py [hunk 4]
```diff
@@ -24,27 +24,71 @@ def __init__(self, config):
...
-        self.auto = False
+        self.calibrated = False
+        self.is_recording = False
+        self.is_autonomous = False
+        self.recv_timestamp = derp.util.get_timestamp()
 
         # Prepare buffers and status variables
-        self.status = None
-        self.last_status = None
+        self.status = {
```
[the remark this answers is not in the document]
Comment (notkarol) [reply derp/joystick.py]: Done.

**Comment by notkarol** [comment]
@John-Ellis Updated to reflect comments.

File: install.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (John-Ellis) [reply install.sh]: Ah good point!

**Pull request merged**